### PR TITLE
Enable video playback as it's somewhat stable now

### PR DESCRIPTION
**Pull request by X1nto**

### CODE REVIEW

**Comment by DiamondMiner88** [comment]
It works pretty well however the controls are really out of place (ugly) so we would need to replace those if possible

**Comment by DiamondMiner88** [comment]
Also, it seems like while the player is loading it doesn't have the proper dimensions and shifts layout once its loaded

**Comment by X1nto** [comment]
> It works pretty well however the controls are really out of place (ugly) so we would need to replace those if possible

We have to either wait for them to release an official compose integration or we can just hide original controls and overlay ours on top

> Also, it seems like while the player is loading it doesn't have the proper dimensions and shifts layout once its loaded

Yep, noticed that. Can't seem to find a fix for it though.

**Review by zt64 — APPROVED** [review]

File: app/src/main/java/com/xinto/opencord/ui/screen/Chat.kt [hunk 1]
```diff
@@ -326,6 +326,9 @@ private fun ChatScreenLoaded(
                                                 height = attachment.height
                                             )
                                         }
+                                        is DomainAttachment.Video -> {
+                                            WidgetAttachmentVideo(url = attachment.url)
```
Comment (DiamondMiner88) [inline app/src/main/java/com/xinto/opencord/ui/screen/Chat.kt:330]: Is there no way to provide a static size so it doesn't resize when done buffering?

Comment (X1nto) [reply app/src/main/java/com/xinto/opencord/ui/screen/Chat.kt:330]: It would be possible if I knew how to convert video dimensions to compose size

Comment (DiamondMiner88) [reply app/src/main/java/com/xinto/opencord/ui/screen/Chat.kt:330]: Afaik discord still sends size params to video attachments, stable does it somehow

**Pull request merged**